### PR TITLE
Feature/sig 529 coverage naar 92 procent

### PR DESCRIPTION
**Pull request by jasperswart**

### CODE REVIEW

File: src/components/MainMenu/index.test.js [hunk 1]
```diff
@@ -1,32 +1,53 @@
...
+
+      expect(wrapper).toMatchSnapshot();
+    });
   });
 
-  it('should render render 2 NavLink components when not authenticated', () => {
-    const wrapper = createComponent();
-    expect(wrapper.find('NavLink').length).toEqual(1);
+  describe('events', () => {
+    it('should render render 1 NavLink components when not authenticated', () => {
```
Comment (RobertLodder) [inline src/components/MainMenu/index.test.js]: Not describing what is tested

File: src/shared/services/auth/auth.test.js [hunk 2]
```diff
@@ -278,4 +294,101 @@ describe('The auth service', () => {
...
+      savedAccessToken = '123AccessToken';
+      initAuth();
+      const name = getName();
+
+      expect(name).toEqual('');
+    });
+
+    it('should return the scopes', () => {
+      parseAccessToken.mockImplementation(() => ({
+        name: 'Jezus Christus'
```
Comment (RobertLodder) [inline src/shared/services/auth/auth.test.js]: Maybe not the best test name for a public repository

File: src/signals/incident/components/IncidentPreview/index.test.js [hunk 3]
```diff
@@ -1,10 +1,71 @@
...
-  it('Expect to have unit tests specified', () => {
-    expect(true).toEqual(true);
+  let props;
+
+  beforeEach(() => {
+    props = {
+      incidentContainer: {
+        incident: {
+          phone: '0666 666 666',
+          email: 'devil@hell.com'
```
Comment (RobertLodder) [inline src/signals/incident/components/IncidentPreview/index.test.js]: Not the best test email for a public repository

File: src/signals/incident/containers/IncidentContainer/index.test.js [hunk 4]
```diff
@@ -1,10 +1,61 @@
...
+      expect(wrapper).toMatchSnapshot();
+    });
+  });
+
+  describe('mapDispatchToProps', () => {
+    const dispatch = jest.fn();
+
+    it('should get classification', () => {
+      mapDispatchToProps(dispatch).getClassification('alweer poep');
+      expect(dispatch).toHaveBeenCalledWith({ type: GET_CLASSIFICATION, payload: 'alweer poep' });
```
Comment (RobertLodder) [inline src/signals/incident/containers/IncidentContainer/index.test.js]: Not a suitable payload 😅

File: src/shared/services/auth/auth.test.js (hunk 2 above)
Comment (jasperswart) [reply src/shared/services/auth/auth.test.js]: gedaan

File: src/components/MainMenu/index.test.js (hunk 1 above)
Comment (jasperswart) [reply src/components/MainMenu/index.test.js]: gedaan

File: src/signals/incident/components/IncidentPreview/index.test.js (hunk 3 above)
Comment (jasperswart) [reply src/signals/incident/components/IncidentPreview/index.test.js]: gedaan

**Pull request merged**